react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -401,10 +444,12 @@ function _mapStateToProps(state, ownProps) {
         _isModerator: Boolean(localParticipant?.role === PARTICIPANT_ROLE.MODERATOR),
         _disableKick: Boolean(disableKick),
         _disableRemoteMute: Boolean(disableRemoteMute),
+        _filmstripHeight: filmstripHeight,
         _remoteControlState,
         _menuPosition,
         _overflowDrawer: overflowDrawer,
         _participantDisplayName,
+        _participantCount: participantCount,
```

Comment:
Maybe having `participantCount ===2` as a prop instead of `participantCount` will be more efficient. WDYT?

Reply:
The styles for the connection status popover are updating base on the number present in the conference. 
I think it's important to follow it.

Reply:
What do you mean? I see the only usage of participantCount as `this.props._participantCount === 2`.

Or do you mean that if somebody joins/leave we need to recalculate something totally different? If so what is it?

Reply:
Different styles are applied based on the participantCount and that is why we need to follow it update.